docs/API.md

```diff
@@ -52,6 +52,14 @@ A method returning a `ReactTestInstance` with matching text – may be a string
 
 A method returning an array of `ReactTestInstance`s with matching text – may be a string or regular expression.
 
+### `getByPlaceholder: (placeholder: string | RegExp)`
+
+A method returning a `ReactTestInstance` for a `TextInput` with a matching placeholder – may be a string or regular expression. Throws when no matches.
+
+### `getAllByPlaceholder: (text: string | RegExp)`
```

Comment:
Can we change this `text` to `placeholder`? Or the other way around if you feel like `text` is nicer :)
```suggestion
### `getAllByPlaceholder: (placeholder: string | RegExp)`
```

Reply:
Well crap, I thought I grabbed all of them!